.github/workflows/docker.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
```

Comment:
For situations with CookRight, we have a multiarch build which requires a different set of instructions.